backend/eth/client_test.go

```diff
@@ -123,7 +123,7 @@ func TestGetExpiryInvalidAddress(t *testing.T) {
 		Address: "0x",
 	})
 	assert.Error(t, err)
-	assert.Equal(t, "[2006] error code InputError with desc Provided invalid address.", err.Error())
+	assert.Equal(t, "[2006] error code InputError with desc Provided invalid address. with cause invalid address 0x", err.Error())
```

Comment:
As an aside (no change needed): This check makes for a brittle test, as you experienced with this PR. I'd probably change this to something like `assert.Regex(t, "\b2006\b.*\b0x\b")`, just enough to ensure that the main error code and the context are being reported. The exact wording may change later, and that's OK.